Bottom: 47b2a93e911b6e017fa8e0683c7ffbd2dcf8510c
Top:    c6f4c2828412102a8ec5cf6fdc4be0df2beadd4f
Author: Ian Kent <raven@themaw.net>
Date:   2013-07-30 15:41:13 +0800

Refresh of autofs-5.0.7-add-dumpmap-to-file.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index e724ca3..4a488e7 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2177,6 +2177,7 @@ int main(int argc, char *argv[])
 
 		lookup_nss_read_master(master_list, 0);
 		master_show_mounts(master_list, dump);
+		free(dump);
 		exit(0);
 	}
